### PR TITLE
Nsv13 boarding changes

### PR DESCRIPTION
**Pull request by DuskLight9978**
## About The Pull Request
This PR makes changes to all syndicate boarding maps to make them more viable for ghost roles. Such as fighters for the Syndicate crew on the carrier, Medkits for the NT prisoner on the mako. Better turret placement. Reduced Health on the Barricade for the turret inside the Hellfire map. More maint spawns for loot. More syndicate docs for incentivizing boarding. Alongside many smaller changes.

## Why It's Good For The Game
Maintaining the boarding maps is good.

## Testing Photographs and Procedure
board_test used to load boarding maps and verify changes were consistent and not erroneous.
Bokkiewoki said not to add pictures, but to add pictures of spiderman so here you go.

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
tweak: Tweaked the boarding maps.
fix: Added lights to Mako maps
/:cl:

### CODE REVIEW

**Comment by DuskLight9978** [comment]
Excellent, Green checks. Now for the taste test of quality.

File: _maps/templates/boarding/syndicate/mako_carrier.dmm [hunk 1]
```diff
@@ -28,8 +28,14 @@
 /area/ruin/powered/nsv13/boarding_interior)
 "aV" = (
 /obj/structure/displaycase,
+/obj/item/documents/syndicate,
 /turf/open/floor/carpet/red,
 /area/ruin/powered/nsv13/boarding_interior)
+"aW" = (
+/obj/structure/closet/crate,
+/obj/effect/spawner/lootdrop/ruinloot/medical,
```
Comment (Bokkiewokkie) [inline _maps/templates/boarding/syndicate/mako_carrier.dmm:36]:
```suggestion
/obj/effect/spawner/lootdrop/maintenance/six,
```
the medical lootdrop has some pretty gamer equipment, it's probably better to just use maintenance ones for these (and maybe add a single medical ruin loot spawner in the medbay?)
Images for reference:
image
image

File: _maps/templates/boarding/syndicate/mako_carrier.dmm [hunk 2]
```diff
@@ -603,6 +655,13 @@
 /obj/machinery/computer/ship/viewscreen,
 /turf/open/floor/carpet/red,
 /area/ruin/powered/nsv13/boarding_interior)
+"Eb" = (
+/obj/structure/closet/crate,
+/obj/effect/spawner/lootdrop/ruinloot/medical,
+/obj/effect/spawner/lootdrop/ruinloot/medical,
+/obj/effect/spawner/lootdrop/ruinloot/medical,
```
Comment (Bokkiewokkie) [inline _maps/templates/boarding/syndicate/mako_carrier.dmm:662]:
```suggestion
/obj/effect/spawner/lootdrop/maintenance/six,
```
Ditto regarding the ruin loot, you could have one or two in there but this is probably a bit much (seeing as it's also two crates)

File: _maps/templates/boarding/syndicate/mako.dmm [hunk 3]
```diff
@@ -462,13 +502,26 @@
...
 "zq" = (
 /obj/structure/peacekeeper_barricade/metal{
 	dir = 1
 	},
 /obj/item/stack/barbed_wire/five,
 /turf/open/floor/plating,
 /area/ruin/powered/nsv13/boarding_interior)
+"zG" = (
+/obj/structure/closet/crate,
+/obj/effect/spawner/lootdrop/ruinloot/medical,
```
Comment (Bokkiewokkie) [inline _maps/templates/boarding/syndicate/mako.dmm:522]: Same notes as with the mako carrier regarding these
```suggestion
/obj/effect/spawner/lootdrop/maintenance/six,
```

File: _maps/templates/boarding/syndicate/mako.dmm [hunk 4]
```diff
@@ -656,6 +725,13 @@
 	},
 /turf/closed/wall/r_wall,
 /area/ruin/powered/nsv13/boarding_interior)
+"LZ" = (
+/obj/structure/closet/crate,
+/obj/effect/spawner/lootdrop/ruinloot/medical,
+/obj/effect/spawner/lootdrop/ruinloot/medical,
+/obj/effect/spawner/lootdrop/ruinloot/medical,
```
Comment (Bokkiewokkie) [inline _maps/templates/boarding/syndicate/mako.dmm:732]:
```suggestion
/obj/effect/spawner/lootdrop/maintenance/six,
```
Ditto

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]

**Review by Bokkiewokkie — APPROVED** [review]

**Pull request merged**